scipy/io/wavfile.py

```diff
@@ -742,10 +742,11 @@ def write(filename, rate, data):
     Write to 16-bit PCM, Mono.
 
     >>> from scipy.io.wavfile import write
+    >>> import numpy as np
     >>> samplerate = 44100; fs = 100
     >>> t = np.linspace(0., 1., samplerate)
     >>> amplitude = np.iinfo(np.int16).max
-    >>> data = amplitude * np.sin(2. * np.pi * fs * t)
+    >>> data = np.int16(amplitude * np.sin(2. * np.pi * fs * t))
     >>> write("example.wav", samplerate, data.astype(np.int16))
```

Comment:
Isn't `astype` here doing the same thing already?

Reply:
It certainly is. I'm not sure how I missed that initially. Sorry for my confusion.